pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "genome-browser"
+version = "0.1.1"
+description = "A lightweight Python 3 graphing API for genomic graphs over genomic intervals."
+authors = ["Clint Valentine <valentine.clint@gmail.com>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
+matplotlib = "^3.5.2"
```

Comment:
![168710268-7be6e8f8-6dca-4031-9038-ae6fe666da85](https://user-images.githubusercontent.com/13955209/168710780-7bd050fe-a1ca-42ce-8be4-f4dc372525ed.png)

Same for the three other libs below.